meta-ostro/conf/distro/ostro.conf

```diff
@@ -1,6 +1,12 @@
 DISTRO = "ostro"
 DISTRO_NAME = "Ostro OS"
-DISTRO_VERSION = "1.0~snapshot-${DATE}"
+# The version is bumped after each official release.
+# This implies that new releases go out with a version
+# string based on the predecessor plus the suffix, which
+# was deemed sufficient in the "[Ostro-dev] versioning Ostro OS"
```

Comment:
How about adding a link to it, i.e.: https://lists.ostroproject.org/pipermail/ostro-dev/2016-May/000427.html ?